components/profile/OccupationSelect.tsx

```diff
@@ -19,24 +19,25 @@ export const OccupationSelect = ({
     "looking-for-job": "Hledám práci",
     "other": "Jiné",
```

Comment:
Navrhuju ještě v rámci téhle změny zrušit možnost _Jiné_, s těmi checkboxy už mně definitivně nedává smysl :) V databázi projdu a zahodím.

Reply:
Smáznu.